components/server/src/init.ts

```diff
@@ -4,6 +4,19 @@
  * See License.AGPL.txt in the project root for license information.
  */
 
+if (process.env.ENABLE_GOOGLE_CLOUD_PROFILER === "true") {
```

Comment:
@geropl Would it be alright if i feature flag it? 

In case if we have performance issues we will disable and recycle servers. I think it is better than installer config.

Unfortunately it does not stop dynamic stopping: https://github.com/googleapis/cloud-profiler-nodejs/issues/876

Reply:
Yeah, SGTM!